native/src/components/NavigationTiles.tsx

```diff
@@ -62,7 +62,7 @@ class NavigationTiles extends React.PureComponent<PropsType, StateType> {
     const contentSizeDiff = nativeEvent.contentSize.width - nativeEvent.layoutMeasurement.width
     this.setState({
       xPosition: nativeEvent.contentOffset.x,
-      contentSizeDiff: parseInt(contentSizeDiff.toFixed(0))
+      contentSizeDiff: parseInt(contentSizeDiff.toFixed(0), 10)
```

Comment:
Why is radix needed? It is optional.

Reply:
Yes it is, but from the docs:
> An integer between 2 and 36 that represents the radix (the base in mathematical numeral systems) of the string. Be careful—this does not default to 10! If the radix value is not of the Number type it will be coerced to a Number.

We could also disable the rule oc but I think it makes sense (or does not hurt at least). The corresponding rule is https://eslint.org/docs/rules/radix
> By default, parseInt() will autodetect decimal and hexadecimal (via 0x prefix). Prior to ECMAScript 5, parseInt() also autodetected octal literals, which caused problems because many developers assumed a leading 0 would be ignored.

This confusion led to the suggestion that you always use the radix parameter to parseInt() to eliminate unintended consequences. So instead of doing this:

Reply:
yeah, oke, makes sense.